cibuildwheel/windows.py

```diff
@@ -421,7 +421,7 @@ def build(options: BuildOptions) -> None:
                 shell(test_command_prepared, cwd="c:\\", env=virtualenv_env)
 
                 # clean up
-                shutil.rmtree(venv_dir)
+                shutil.rmtree(venv_dir, ignore_errors=True)
```

Comment:
```suggestion
                # ignore_errors: because sometimes this fails because another process is using the files. 
                # See #825
                shutil.rmtree(venv_dir, ignore_errors=True)
```